todo_app/session_items.py

```diff
@@ -0,0 +1,69 @@
+from flask import session
```

Comment:
Delete this file. It's just a duplicate of the other session_items.py and this one is unused.